packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -572,6 +581,24 @@ export abstract class SharedObjectCore<
 		this.emitInternal("op", message, local, this);
 	}
 
+	/**
+	 * Process messages for this shared object. The messages here are contiguous messages for this object in a batch.
+	 * @param messagesWithMetadata - A bunch of contiguous messages for this object and their metadata.
+	 * @param local - true if the messages were originally generated by the client receiving it.
+	 */
+	private processMessages(
```

Comment:
should this be protected so that dds can override it

Reply:
I was thinking of doing something with it in a follow up. Basically, have a `processMessagesCore` similar to `processCore` that a DDS could implement.